sklearn/compose/tests/test_column_transformer.py

```diff
@@ -342,19 +342,15 @@ def test_make_column_transformer_kwargs():
     scaler = StandardScaler()
     norm = Normalizer()
     ct = make_column_transformer(('first', scaler), (['second'], norm),
-                                 n_jobs=3)
+                                 n_jobs=3, remainder='drop',
+                                 transformer_weights={'first': 0.5,
+                                                      'second': 1.5})
     assert_equal(
         ct.transformers,
         make_column_transformer(('first', scaler),
                                 (['second'], norm)).transformers)
     assert_equal(ct.n_jobs, 3)
-    # invalid keyword parameters should raise an error message
-    assert_raise_message(
-        TypeError,
-        'Unknown keyword arguments: "transformer_weights"',
-        make_column_transformer, ('first', scaler), (['second'], norm),
-        transformer_weights={'pca': 10, 'Transf': 1}
-    )
+    assert_equal(ct.remainder, 'drop')
```

Comment:
also add a similar check for `transformer_weights` ?

Reply:
ups